clients/ui/frontend/src/app/pages/modelCatalog/components/ValidatedModelCard.tsx

```diff
@@ -0,0 +1,247 @@
+              <Skeleton
+                shape="square"
+                width="56px"
+                height="56px"
+                screenreaderText="Brand image loading"
+                style={{ marginRight: '8px' }}
+              />
+            )}
+            <FlexItem align={{ default: 'alignRight' }}>
+              <Badge className="pf-m-purple">Validated</Badge>
+            </FlexItem>
+          </Flex>
+          <div style={{ marginTop: '8px' }}>
+            <Flex alignItems={{ default: 'alignItemsCenter' }} gap={{ default: 'gapSm' }}>
+              <div
+                data-testid="validated-model-status-indicator"
+                className={text.textColorStatusSuccess}
+                style={{
+                  width: '8px',
+                  height: '8px',
+                  backgroundColor: 'var(--pf-t--global--palette--green-400)',
+                  borderRadius: '50%',
+                }}
+              />
+              <Content component={ContentVariants.small} className={text.textColorStatusSuccess}>
+                Success
+              </Content>
+            </Flex>
+            <Link to={catalogModelDetailsFromModel(model.name, source?.id)}>
+              <Button
+                data-testid="model-catalog-detail-link"
+                variant="link"
+                tabIndex={-1}
+                isInline
+                style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
+              >
+                {truncate ? (
+                  <span>{getModelName(model.name)}</span>
+                ) : (
+                  <span>{getModelName(model.name)}</span>
+                )}
+              </Button>
+            </Link>
+          </div>
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
+        <div>
+          <div style={{ paddingLeft: '8px', marginBottom: '16px' }}>
+            <Flex
+              alignItems={{ default: 'alignItemsCenter' }}
+              justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            >
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <ChartBarIcon />
+                <span
+                  data-testid="validated-model-accuracy"
+                  style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
+                >
+                  {metrics.accuracy}%
+                </span>
+              </Flex>
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <span className="pf-v5-c-content pf-m-small">Average accuracy</span>
+                <Tooltip
+                  content="Average accuracy
+
+The weighted average of normalized scores from all benchmarks. Each benchmark is normalized to a 0-100 scale. All normalized benchmarks are then averaged together."
```

Comment:
The whitespace in this content gets collapsed to a space. ~~You can pass JSX to the tooltip's content prop and use that to render this as a fragment `<></>` with some line breaks in it `<br /><br />` so "Average accuracy" appears separately.~~

Actually, looking closer at the mockups this should be a Popover (appears on click) and not a Tooltip, and "Average accuracy" should be used as the popover title. Same with the Latency help text.

<img width="904" height="470" alt="Screenshot 2025-10-07 at 5 17 19 PM" src="https://github.com/user-attachments/assets/9263b4bd-cca9-4c8a-afa5-9eb06b536326" />